bndtools.core/bndtools.shared.bndrun

```diff
@@ -175,7 +182,7 @@ crossplatform.bundles: \
 	${help.bundles},\
 	${p2.bundles},\
 	${pde.bundles},\
-	${m2e.bundles},\
+	${m2e.debug.bundles},\
```

Comment:
I don't think this is something to commit to the normal build. It is only for m2e debugging purposes. See my comment on the p2 repo with the m2e debug bundles.